spk/mono/src/service-setup.sh

```diff
@@ -0,0 +1,15 @@
+sync_certificates ()
+{
+    # Sync certificate
+    ${SYNOPKG_PKGDEST}/bin/cert-sync /etc/ssl/certs/ca-certificates.crt >> ${INST_LOG} 2>&1
+}
+
+service_postinst ()
+{
+    sync_certificates
+}
+
+service_postupgrade ()
+{
+    sync_certificates
```

Comment:
Why twice? Since it's already done during `postinst`?

Reply:
Right. I just copied it from previous installer.sh